### PR TITLE
Change clientset to use config from config.GetConfig()

### PR DESCRIPTION
**Pull request by Janek3d**
During debugging in VS Code on cluster deployed on AWS, I was getting error that GetClientset function could not locate kubeconfig file despite providing "kubeconfig" flag and "KUBECONFIG" variable in launch.json file. After some digging I found that GetClientset uses getClientConfig function which was simply ignoring this flag and variable. I decided to get clientsert by using more direct function GetClientsetFromConfig and config gathered by config.GetConfig. I also deleted GetConfig function which was not used anywhere.

### CODE REVIEW

**Review by Dysproz — APPROVED** [review]
Great input Janusz!

File: cmd/manager/main.go [hunk 1]
```diff
@@ -91,7 +91,7 @@ func main() {
 		os.Exit(1)
 	}
 
-	clientset, err := v1alpha1.GetClientset()
+	clientset, err := v1alpha1.GetClientsetFromConfig(cfg)
```
Comment (kaweue) [inline cmd/manager/main.go:94]: In my opinion you can also delete `GetClientsetFromConfig` and move `clientset, err := kubernetes.NewForConfig(config)` here. I really don't see a point of having a function that only calls one other function.

**Review by kaweue — CHANGES_REQUESTED** [review]

File: cmd/manager/main.go (hunk 1 above)
Comment (Janek3d) [reply cmd/manager/main.go]: I changed it to `kubernetes.NewForConfig(cfg)`.

**Review by kaweue — APPROVED** [review]

**Pull request merged**